doc/usage/configuration.rst

```diff
@@ -3046,8 +3046,11 @@ These options influence LaTeX output.
          Please update your project to use the
          :ref:`latex table color configuration <tablecolors>` keys instead.
 
-   Each table can override the global style via ``:class:`` option,
-   or ``.. rst-class::`` for no-directive tables (cf.  :ref:`table-directives`).
+   To customize the styles on an individual table, either set the ``:class:``
+   option when the table's contents are documented inside a table directive,
+   or, otherwise, add a preceding :ref:`rst-class <rstclass>` directive
+   (cf. :ref:`table-directives`).
```

Comment:
```suggestion
   To customize the styles on an individual table, either set the ``:class:``
   option on the enclosing table directive - if there is one - or insert a
   :ref:`rst-class <rstclass>` directive before the table otherwise
   (cf. :ref:`table-directives`).
```